File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Models/TableModel/MeetingRecord.cs
namespace MODERN_ART_ASSOCIATION_OF_KAOHSIUNG.Models.TableModel
{
    public class MeetingRecord
    {
        /// <summary>
        /// 資料表名稱
        /// </summary>
        public string TABLE_NAME { get { return "MeetingRecord"; } }
        
        /// <summary>
        /// 會議記錄日期
        /// </summary>
        public string MeetingRecord_Date { get; set; }

        /// <summary>
        /// 會議記錄詳細資料
        /// </summary>
        public string MeetingRecord_Detail { get; set; }

        /// <summary>
        /// 會議記錄結束時間
        /// </summary>
        public string MeetingRecord_Endtime { get; set; }

        /// <summary>
        /// 檔案路徑
        /// </summary>
        public string MeetingRecord_File { get; set; }

        /// <summary>
        /// 會議紀錄編號
        /// </summary>
        public string MeetingRecord_ID { get; set; }

        /// <summary>
        /// 修改時間
        /// </summary>
        public string MeetingRecord_Modificationdatetime { get; set; }

        /// <summary>
        /// 會議地點
        /// </summary>
        public string MeetingRecord_Place { get; set; }

        /// <summary>
        /// 會議記錄開始時間
        /// </summary>
        public string MeetingRecord_Starttime { get; set; }

        /// <summary>
        /// 會議紀錄標題
        /// </summary>
        public string MeetingRecord_Title { get; set; }

        public void PK_MeetingRecord(string MeetingRecord_ID)
        {

        }
    }
}

File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Scripts/Backstage/CF/MeetingRecord.js
$(document).ready(function () {
    $.ajax({
        url: '/BCF/GetMeetingRecorddata_B',
        type: 'POST',
        dataType: 'json',
        success: function (data) {
            console.log(data);
        },
        error: function () {
            console.log("無法取得db");
        }
    });
    //$.ajax({
    //    url: '/BCF/CreateMeetingRecorddata_B',
    //    type: 'POST',
    //    data: {
    //        MeetingRecord_Title: 'test333',
    //        MeetingRecord_Detail: 'test',
    //        MeetingRecord_Date: '2015/08/30',
    //        MeetingRecord_Starttime: '10:00:00',
    //        MeetingRecord_Endtime: '11:00:00',
    //        MeetingRecord_Place: 'test',
    //        MeetingRecord_File: 'test',
    //        MeetingRecord_Modificationdatetime: '11:00:00'
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BCF/UpdateMeetingRecorddata_B',
    //    type: 'POST',
    //    data: {
    //        MeetingRecord_Title: 'test333456',
    //        MeetingRecord_Detail: 'test111',
    //        MeetingRecord_Date: '2015/08/31',
    //        MeetingRecord_Starttime: '11:00:00',
    //        MeetingRecord_Endtime: '12:00:00',
    //        MeetingRecord_Place: 'test111',
    //        MeetingRecord_File: 'test111',
    //        MeetingRecord_Modificationdatetime: '13:00:00',
    //        MeetingRecord_ID: 6
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BCF/DeleteMeetingRecorddata_B',
    //    type: 'POST',
    //    data: {
    //        MeetingRecord_ID: 6
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
})